templates/filesystem/Assets_HTAccess.ss

```diff
@@ -0,0 +1,27 @@
+#
+# Whitelist appropriate assets files.
+# This file is automatically generated via File.allowed_extensions configuration
+# See AssetAdapter::renderTemplate() for reference.
+#
+
+<IfModule mod_rewrite.c>
+    SetEnv HTTP_MOD_REWRITE On
+    RewriteEngine On
+
+    # Disable PHP handler
+    RewriteCond %{REQUEST_URI} .(?i:php|phtml|php3|php4|php5|inc)$
+    RewriteRule .* - [F]
+
+    # Allow error pages
+    RewriteCond %{REQUEST_FILENAME} -f
+    RewriteRule error[^\\/]*\.html$ - [L]
+
+    # Block invalid file extensions
+    RewriteCond %{REQUEST_URI} !\.(?i:<% loop $AllowedExtensions %>$Extension<% if not $Last %>|<% end_if %><% end_loop %>)$
```

Comment:
Oh, actually, I am escaping it anyway. :) If you check AssetAdapter.php you'll see I'm doing preg_quote.

![image](https://cloud.githubusercontent.com/assets/936064/11946031/18abf780-a8bb-11e5-9b08-4e4d5ed0c3b2.png)